shumway/__init__.py

```diff
@@ -51,15 +52,19 @@ def __init__(self, what, key, attributes=None, tags=None, value=0):
     def update(self, value):
         self.value = value
 
-    def flush(self, func):
-        """Create a map of data and pass it to another function"""
-        func({
+    def as_map(self):
```

Comment:
Is this expected to be a public method? It should be added to the readme if so, or moved to a `_as_map` private method otherwise.

Reply:
I renamed it to as_dict since that seem to be more aligned with python and added docs